components/scream/src/physics/zm/zm_inputs_initializer.cpp

```diff
@@ -198,13 +197,38 @@ void ZMInputsInitializer::add_field (const field_type &f)
   m_fields_id.insert(id);
 }
 
+void ZMInputsInitializer::
```

Comment:
Presumably we would need to create something like this for SHOC and RAD right?  Would it be simpler to make the Initializer a general class or object so that if we change the code we don't have to make the same change in 4+ places?

Reply:
I'm not sure what you mean. There is already a FieldInitializer class, so that the AD can interact in a process-agnostic way.